DigitalOffice/Tests/ProjectServiceTests/Commands/EditProjectByIdCommandTests.cs

```diff
@@ -0,0 +1,157 @@
+﻿using Moq;
+using System;
+using NUnit.Framework;
+using FluentValidation;
+using LT.DigitalOffice.ProjectService.Models;
+using LT.DigitalOffice.ProjectService.Commands;
+using LT.DigitalOffice.ProjectService.Database.Entities;
+using LT.DigitalOffice.ProjectService.Mappers.Interfaces;
+using LT.DigitalOffice.ProjectService.Commands.Interfaces;
+using LT.DigitalOffice.ProjectService.Repositories.Interfaces;
+
+namespace LT.DigitalOffice.ProjectServiceUnitTests.Commands
+{
+    public class EditProjectByIdCommandTests
+    {
+        private Guid projectId;
+
+        private DbProject dbProject;
+
+        private EditProjectRequest editRequest;
+
```

Comment:
Lots of extra blank lines

Reply:
How the H are they extra? Its all subjective. I disagree with such comments.

In my opinion, these kind of variables should be declared separately.

Extra blank space or not, it is all readable and working. Its not even typo or naming kind of issues.

Also: "a lot of..." so there's more? Where exactly? Please address them because I can't seem to keep track of all unnecessary changes I need to make.

Reply:
Also these kind of comments should be fixed in the separate Refactoring PR, because if we review like that it takes million years to close the PR.

Soooooooooo much wasted time and yours and mine. 

This should be left to Refactoring PR because in develop THERE ARE ALREADY a lot of "mistakes" and "incorrect names" etc. 

So this is a re-occurring issue. Yea I've messed up names or spaces here and there. But this is in no way related to readability, workability or everything useful in this code.
Hell, one man can view a PR a couple of times and each time be unhappy with SOMETHING.
So it already indicates that this should be a different PR.

God I'm so sick of making useless changes and arguing which name is better, how to write "cooler" because this is all subjectivity and on any new working place names and spacing will be different.  
I'm here to learn techniques and technologies, and I seem to learn nothing more during the past 2 or 3 weeks. This is disappointing because a lot of time wasted for nothing. Nothing important at least.

Reply:
I'd rather myself start refactoring all code but I'd do it in a single day knowing that a lot of was done and it all needs to look cool, but all I have to do is go through every line of code ONCE. And not be distracted 5 times a day trying to fix one little line of code. Oh, and then fix another line of code but all the changes are COSMETIC.

Reply:
No offense @KostinP , this is addressed to the whole team. I just got sick of all of it.

Reply:
There you go. Everything's smooshed together, enjoy